### DISCUSSION TITLE
Allow for multiselect in layered navigation

### DISCUSSION THREAD

**Discussion by kiatng** [OpenMage/magento-lts#3420] [Ideas]
For example, take the color attribute, currently it's not possible in layered navigation to filter for results with more than one color. There are extensions that allow it but I recently implemented this feature for a client. 

If the community find it useful, I can make a few related PRs:

1. Remove dependency on CatalogIndex in the layered navigation.
2. Improve layered navigation page load speed when there is no active filters. 
3. Multiselect in layered navigation.

**Comment by fballiano** [comment]
I think it'd be extremely useful

**Comment by addison74** [comment]
It is fine to filter a category using several values of an attribute. I think I used a free extension at one time, but if you have a personal solution I have no problem with that. It would be good to have an option to activate/deactivate this feature, especially for those who are used to only one value.

Also, in the past we had a discussion on this topic related to Layering Navigation. There are situations when certain attributes must be hidden. This could be solved in Backend in the category page, with a section in which to choose the attributes to be skipped from the listing in LN. More information here https://magento.stackexchange.com/questions/102184/hide-an-attribute-in-layering-navigation-for-a-specified-category.

**Comment by addison74** [comment]
This is the extension I used https://www.manadev.com/magento-extensions/advanced-layered-navigation-ce.
